Q2.cpp

```diff
@@ -0,0 +1,83 @@
+//CodeU: Q2
+//
+//  Created by Julia Semavina on 18.05.2017
+//
+//  problem specification:
+//      Implement an algorithm to find the kth to last element of a singly linked list
+//
+//  comments:
+//      list length is unknown
+//      if K is bigger than the size of the list the answer will be IMPOSSIBLE INPUT
+//      KthToLast function contains a solution
+//
+
+#include <iostream>
+#include <vector>
+
+
+struct Node {
+    char value;
```

Comment:
Not a big deal, and not part of the problem, but it's nicer to use a templated type:
template <typename T>
struct Node {
     T value;
     Node* next;
 };